Tests/UnitTests/FileShareService.DesktopClientTests/Modules/Admin/AdminViewModelTests.cs

```diff
@@ -20,8 +20,8 @@ public class AdminViewModelTests
         private IKeyValueStore fakeKeyValueStore = null!;
         private IJobsParser fakeJobsParser = null!;
         private IEnvironmentsManager fakeEnvironmentsManager = null!;
-        private readonly ILogger<AdminViewModel> logger;
-        private readonly ILogger<NewBatchJobViewModel> Nlogger;
+        private ILogger<AdminViewModel> fakeLoggerAdminVM;
+        private ILogger<NewBatchJobViewModel> fakeLoggerNewBatchVM;
```

Comment:
Please address the compiler warnings:
````
Non-nullable field 'fakeLoggerAdminVM' must contain a non-null value when exiting constructor. Consider declaring the field as nullable.
Non-nullable field 'fakeLoggerNewBatchVM' must contain a non-null value when exiting constructor. Consider declaring the field as nullable.
````

Reply:
Resolved